Services/Simulations.cs

```diff
@@ -91,7 +92,7 @@ public async Task<IList<Models.Simulation>> GetListAsync()
                 result.Add(simulation);
             }
 
-            return result;
+            return result.OrderByDescending(s => s.Modified).ToList();
```

Comment:
TODO: eventually this will return a long list, potentially paginated, so we need to discuss how to get the latest ones first, or paginate etc.